### PR TITLE
Add styles to experimental-amp example

### PR DESCRIPTION
**Pull request by ijjk**
After discussion added global styles to experimental-amp example

### CODE REVIEW

File: examples/experimental-amp/pages/_app.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    return { pageProps }
+  }
+
+  render () {
+    const { Component, pageProps } = this.props
+
+    return (
+      <Container>
+        <Component {...pageProps} />
+        <style jsx global>{`
```
Comment (timneutkens) [inline examples/experimental-amp/pages/_app.js]: Let's add the styles inside the page, possibly inside a `Layout` component.

Comment (ijjk) [reply examples/experimental-amp/pages/_app.js]: Sounds good, updated

File: examples/experimental-amp/components/Layout.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export default function Layout ({ children }) {
+  return (
+    <>
+      {children}
+      <style jsx global>{`
+        body {font-family: Roboto, sans-serif; padding: 30px; color: #444;}
+        h1 {margin-bottom: 5px;}
```
Comment (timneutkens) [inline examples/experimental-amp/components/Layout.js]: Let's make these styles local, only body is really global.

Comment (ijjk) [reply examples/experimental-amp/components/Layout.js]: Updated

**Review by timneutkens — APPROVED** [review]

**Pull request merged**